app/src/main/java/com/cmput301f20t21/bookfriends/ui/component/ProgressButton.java

```diff
@@ -97,4 +104,53 @@ public void onError() {
 
         view.setClickable(true);
     }
+
+    // book functions for later on
+    public static class BaseBookListAdapter extends RecyclerView.Adapter<BaseBookListAdapter.ViewHolder> {
```

Comment:
I dont think this is supposed to be in ProgressButton.

Reply:
Yes, you are right, it supposes to be put under the component folder